aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -32,6 +32,14 @@ Tensor dequantize_quant(const Tensor& self) {
   return get_qtensorimpl(self)->quantizer()->dequantize(self);
 }
 
+std::vector<Tensor> dequantize_tensors_quant(TensorList tensors) {
+  std::vector<Tensor> dequantized_tensors;
+  for (auto i = 0; i < tensors.size(); ++i) {
```

Comment:
can we use parallel for here?

Reply:
I would delay this optimization until there is a need, most likely this function will never be run since it will be fused into the quantized::cat pattern.